docs/cli-lib/api/getAndLoadConfigIfNeeded.md

```diff
@@ -0,0 +1,35 @@
+## `getAndLoadConfigIfNeeded(options)`
+
+[cli-lib/lib/config.js](https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli-lib/lib/config.js)
```

Comment:
What do folks think about this? At first, I had the link go directly to the function, but the problem with that is if we add code to that file or move anything around, it's going to change the line number that the function starts at, but the permalink will still point at the old line number.   I doubt we would remember (or want to) update every permalink for the file whenever we change it.

This seemed like a decent middle ground.  Linking to the file, so we'd only need to update permalinks if we add/remove/move the function from the file it was in to a different file